### PR TITLE
add in public suffix

### PR DESCRIPTION
**Pull request by hs90095dayan**
we use *.codegardener.com.tw for our projects

### CODE REVIEW

**Comment by weppos** [comment]
Please read https://github.com/publicsuffix/list/wiki/Guidelines and update your request accordingly.

**Comment by hs90095dayan** [comment]
OK we add DNS TXT record 
dig TXT _psl.codegardener.com.tw 
add pull/399

About Email Authentication
I did not receive anything authentication email ? How to get it?

**Comment by weppos** [comment]
Please re-read the guidelines. You need to provide a reason for inclusion, and properly format the patch.

Also, you mentioned the use of subdomains. Can you show some example? I've tried some DNS lookups, and I was not able to find any subdomain.

**Comment by hs90095dayan** [comment]
We are a new station platform
We will provide free domain names to the average user. 

The currently tested subdomain has stage.codegardener.com.tw

**Comment by sleevi** [comment]
I'm proposing to close this as WontFix, as I think the questions in https://github.com/publicsuffix/list/pull/399#issuecomment-279660538 are still outstanding, but I'll give one more chance before doing so.

**Comment by hs90095dayan** [comment]
fix name sort in PRIVATE DOMAINS

we fix name sort in PRIVATE DOMAINS
We will provide the user with a domain name (Is still a development of the site)

some domain example
kencorp.codegardener.com.tw
skysea.codegardener.com.tw

File: public_suffix_list.dat [hunk 1]
```diff
@@ -11806,6 +11806,10 @@ i234.me
 myds.me
 synology.me
 
+// Studio, Green
```
Comment (sleevi) [inline public_suffix_list.dat]: What's the web page address

File: public_suffix_list.dat [hunk 2]
```diff
@@ -11806,6 +11806,10 @@ i234.me
...
+// Submitted by ken <hs90095dayan@hotmail.com>
```
Comment (sleevi) [inline public_suffix_list.dat]: This doesn't seem like a stable e-mail address to contact someone?

**Comment by hs90095dayan** [comment]
hi
because hotmail is my most frequently used email

Another reason is because my whois contact email is hotmail, can you confirm that I need to change email or use email like ken@codegardener.com.tw

thank you for your reply

**Comment by weppos** [comment]
Is this request still valid? @hs90095dayan can you provide also a company URL, and a reason for inclusion?

**Pull request closed without merging**